src/spring-mongo/src/main/java/net/conology/spring/restjsonpath/mongo/NestedValueTestCompiler.java

```diff
@@ -34,10 +32,10 @@ public MongoPropertyCondition compile() {
             var elementMatch = compileElementMatch(nodes);
 
             if (elementMatch == null && nodes.hasNext()) {
-                // field selector should consume everything except for filters that result in elementMatch
+                // field selector should consume everything except for filters that result in
+                // elementMatch
```

Comment:
not really important, but a general advice:

if you line break sentences, try to get at least 4 words on the new line say more, say less, or consider breaking at that. it's just a general editorial style to consider. A linebreak with a single word wastes v space and causes unnecessary cognitive load